test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,25 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() {
+        // Contains at least one null value
+        assertContainsNull((Object) null);
+        assertContainsNull(null, 1, "hello");
+        assertContainsNull(1, null, "bye", null);
+
+        // Does not contain any null value
+        assertDoesNotContainNull("null");
+        assertDoesNotContainNull(1, 1, 1);
+        assertDoesNotContainNull(1, "null", Integer.valueOf(20));
+    }
+
+    private void assertContainsNull(Object... objects) {
+        assertTrue(Utils.isAnyNull(objects));
```

Comment:
Not necessary to abstract the `assertTrue(Utils.isAnyNull(...))` as separate method, because asserting the result of the method itself is fine.

Although existing test `assertAreUnique` does that, but that is to prevent `Arrays.asList(...)` from flying everywhere.